src/canvas/SelectableCanvas.ts

```diff
@@ -647,6 +647,33 @@ export class SelectableCanvas<EventSpec extends CanvasEvents = CanvasEvents>
     this._beforeTransform(e);
   }
 
+  /**
+   * @private
+   * @param {Event} e send the mouse event that generate the finalize down, so it can be used in the event
+   */
+  endCurrentTransform(e?: TPointerEvent) {
+    const transform = this._currentTransform;
+    if (!transform) {
+      return;
+    }
+
+    const { target, action, actionPerformed } = transform;
+    const options = {
+      e,
+      target,
+      transform,
+      action,
+    };
+
+    target.setCoords();
+    this._currentTransform = null;
```

Comment:
this is a change done when sqaushing `_finalizeCurrentTransform` requiring a bit more work to adapt mouse up logic